frontend/src/components/svgs/index.js

```diff
@@ -203,6 +203,22 @@ const InstaIcon = ({ className = "scale-100" }) => (
   </svg>
 );
 
+const Circle = ({ color }) => (
+  <svg
+    width="80"
+    height="80"
+    viewBox="0 0 80 80"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <rect width="80" height="80" rx="40" fill={color} />
```

Comment:
Here is a suggestion for tailwindcss 
```suggestion
    <rect width="80" height="80" rx="40" className={`fill-[${color}]`} />
```